flycheck.el

```diff
@@ -10668,6 +10668,18 @@ When non-nil, pass the directory via the `--workdir' option."
   :package-version '(flycheck . "32"))
 (make-variable-buffer-local 'flycheck-ghdl-workdir)
 
+(flycheck-def-option-var flycheck-ghdl-ieee-library nil vhdl-ghdl
+  "The standard to use for the ieee library
+
+The value of this variable is either a string denoting an ieee library
+standard, or nil, to use the default standard.  When non-nil,
+pass the ieee library standard via the `--ieee' option."
+  :type '(choice (const :tag "Default standard" nil)
+                 (string :tag "IEEE library standard"))
```

Comment:
Is there an exhaustive list of standards?  If so, we could hardcode them to make the customization more useful.

Reply:
Found a list [in ghdl docs](https://ghdl.readthedocs.io/en/latest/using/InvokingGHDL.html#cmdoption-ghdl-ieee). Changed to hardcoded options in latest commit.